lightwood/__init__.py

```diff
@@ -1,3 +1,4 @@
+import torchvision
```

Comment:
why this?

Reply:
This change was made as a hack in order to fix this issue: https://github.com/mindsdb/lightwood/issues/95

We already do this in `mindsdb` in order to not have it crash (since it's easier to update that rather than update lightwood), but this fixes the issue in case we use lightwood independently.

It's a hack, but it works, might not be required once we update to the next torch version, but it's pretty harmless so I'd rather have it in for now.

Forgot I added it to this PR so I didn't explain this in the description, my bad.